classes/class4/hw/views/ingredients.handlebars

```diff
@@ -0,0 +1,22 @@
+<h1>Ingredient List</h1>
+<form id="addForm" action="newIngredient" method= "POST">
+	Name: <input type="text" name="name"/><br/>
+	Price: <input type="text" name="price"/><br/>
+	<input type ="submit" value="Submit">
+</form>
+<ul>
+{{#each ingredients}}
+<div id ="{{this._id}}">
+<form id="ajax-form1" action="outOfStock" method= "POST">
+	<input type="text" name="id" value = "{{this._id}}" readonly/><br/>
```

Comment:
I would put the id into the id attribute of the form or another attribute so that it is not visible to the user.